servers/src/main/java/tachyon/master/file/meta/InodeTree.java

```diff
@@ -429,6 +429,7 @@ public void setPinned(Inode inode, boolean pinned) {
    * @return true if the given file id is the root id
    */
   public boolean isRootId(long fileId) {
+    Preconditions.checkNotNull(mRoot, "Cannot call isRootId() before initializeRoot()");
```

Comment:
Should all precondition messages go in PreconditionMessage.java? What is the convention?